app/src/components/Navbar/ThemeButton.tsx

```diff
@@ -1,40 +1,37 @@
-import { BsBrightnessHigh, BsMoonStars } from 'react-icons/bs'
-import { MdDevices } from 'react-icons/md'
-import useComponentVisible from '../../hooks/useComponentVisible';
-
+import { BsBrightnessHigh, BsMoon } from 'react-icons/bs'
+import { useState, useEffect } from 'react';
 
 const ThemeButton: React.FC = () => {
-    const { ref, isComponentVisible, setComponentVisible } = useComponentVisible(false);    
+    const [theme, setTheme] = useState('');
```

Comment:
Shouldn't this have a default state of light or dark in order to render the component on First Load successfully without any errors.

Reply:
@BradyWaughen

Reply:
I thought it was not best to have a default state set because we were looking at system preferences anyways.

A default state would be ideal when we implement the theme with Firebase.  @dineshUmasankar